### PR TITLE
Add crowns

### PR DESCRIPTION
**Pull request by Ethspresso**
Add designs with crowns, both with and without laser eyes. Include logic to set crown colors, default to golden crown but orange backgrounds get diamond crowns. Crown is also included as a boolean trait.

### CODE REVIEW

**Comment by Ethspresso** [comment]
There's still some tweaking left that I'd like your input on how to achieve:
 * Ensure that genesis cat has a crown
 * Change the logic for crown / no crown, either a different percentage or based on unique ID below a threshold (10k?)

**Comment by hans-crypto** [comment]
Awesome! 🎉🎉 I will review this in some hours! 

FYI @LightRider5

File: src/lib/mooncat-parser.ts [hunk 1]
```diff
@@ -75,6 +75,9 @@ export class MooncatParser {
     // 26 values between: 120 and 145 --> 26/256 --> 0.1015625 --> ~10%
     const orangeBackground = bytes[6] >= 120 && bytes[6] <= 145;
 
+    // 10% chance of crown
+    const crown = bytes[7] >= 120 && bytes[7] <= 145;
```
Comment (hans-crypto) [inline src/lib/mooncat-parser.ts:79]: hmm... I was thinking of making this an off-chain trait. the cat gets a crown if its within a numbered range.
eg. first 1000 cats get a gold crown but only if we click in the image. so every cat will have two images:
- normal image: needs only transactionId + blockId
- 2nd image: unlocks if we know the cat's number

this aligns with the current way of how are collections are done. they are all off-chain.

Comment (hans-crypto) [reply src/lib/mooncat-parser.ts:79]: @Ethspresso what's you opionion about this idea?

**Review by hans-crypto — APPROVED** [review]

File: src/lib/mooncat-parser.ts (hunk 1 above)
Comment (Ethspresso) [reply src/lib/mooncat-parser.ts:79]: Sounds good to me. Will you adjust the code?

Comment (hans-crypto) [reply src/lib/mooncat-parser.ts:79]: yep! 🫡

**Pull request merged**